buildout.cfg

```diff
@@ -1,20 +1,27 @@
 [buildout]
-parts = interpreter test
+extensions = mr.developer
+github = git://github.com/zopefoundation
+github_push = git@github.com:zopefoundation
+extends =
+    https://zopefoundation.github.io/Zope/releases/4.0a6/versions-prod.cfg
+parts =
+    zopepy
+    test
 develop = .
 unzip = true
-index = http://download.zope.org/Zope2/index/2.13.19/
-find-links =
-    http://pypi.python.org/simple/Products.GenericSetup/
-    http://pypi.python.org/simple/Products.PluginRegistry/
-    http://pypi.python.org/simple/five.localsitemanager/
-extends =
-    http://download.zope.org/Zope2/index/2.13.19/versions.cfg
 
-[interpreter]
+auto-checkout =
+
+[sources]
+Zope2 = git ${buildout:github}/Zope pushurl=${buildout:github_push}/Zope
+
+[zopepy]
 recipe = zc.recipe.egg
-interpreter = python
+zopepy = python
```

Comment:
I don't think the egg recipe has a zopepy argument, the ``interpreter = zopepy`` line below is already naming the generated script.